synapse/federation/transport/server/federation.py

```diff
@@ -95,6 +96,20 @@ async def on_PUT(
                 len(transaction_data.get("edus", [])),
             )
 
+            if issue_8631_logger.isEnabledFor(logging.DEBUG):
+                DEVICE_UPDATE_EDUS = {"m.device_list_update", "m.signing_key_update"}
+                device_list_updates = [
+                    edu.content
+                    for edu in transaction_data.get("edus", [])
+                    if edu.edu_type in DEVICE_UPDATE_EDUS
+                ]
+                if device_list_updates:
+                    issue_8631_logger.debug(
+                        "transaction [%s] includes device list updates: %s",
```

Comment:
Do you want to make this log line unique from the other? It looks like this is the receiving side?

Reply:
Ahh, good idea. Thanks!